pyproject.toml

```diff
@@ -19,13 +19,17 @@ requires-python = ">=3.8"
 dynamic = ["version"]
 
 dependencies = [
-    "qiskit~=0.35",
-    "pytket~=1.1",
-    "numpy>=1.21.5,<1.24.0",
-    "mqt.bench~=0.1.4",
-    "pytket-qiskit>=0.24,<0.28",
```

Comment:
is the pytket-qiskit plugin no longer required?

Reply:
It confuses me a little, since it actually is used in the ML Predictor but the installation and tests are green without it being in this file. Do you have any idea why that is the case?

Reply:
Hm. Maybe the `tket` to `qiskit` conversion is a standard feature of `tket` and so the additional plugin is not actually required.

Reply:
I will remove it and see whether the tests fail. If so, I will re-add it.